addons/mrp/models/product.py

```diff
@@ -190,4 +190,6 @@ def action_open_quants(self):
         res = super(ProductProduct, components).action_open_quants()
         if bom_kits:
             res['context']['single_product'] = False
+            if res['context'].get('default_product_tmpl_id', False):
```

Comment:
Nice catch! Thank you @JordiMForgeFlow 

By the way, can you simplify by a single line: `res['context'].pop('default_product_tmpl_id', None)` ?

Reply:
Done! Thank you for the quick response :)